docs/examples/auto-instrumentation/client.py

```diff
@@ -30,13 +31,15 @@
     BatchSpanProcessor(ConsoleSpanExporter())
 )
 
+param_value = sys.argv[1] if len(sys.argv) > 1 else "testing"
```

Comment:
nit -- what do you think about adding a comment about the motivation behind this line?